.github/containerscan/allowedlist.yaml

```diff
@@ -11,10 +11,12 @@ general:
     - CVE-2021-44568
     - CVE-2022-0778
     - CVE-2022-2526
+    - CVE-2022-27664
     # Disputed CVEs
     - CVE-2019-1010022
```

Comment:
Should we take this opportunity to review this allowedList for CVEs?

Reply:
We have ubi image upgrade story, we should check in that story. I will add in acceptance criteria